services/util/util.go

```diff
@@ -456,3 +456,36 @@ func IsStreamToTerminal(stream io.Writer) bool {
 		return false
 	}
 }
+
+// Which attempts to find an executable in PATH and returns its absolute path if successful
+// note that it tries to find the executable anywhere it can, in case of failure to read a directory
+// listed in one of the PATH's components, it will continue its operation and only return an error in case
+// no paths were readable and/or executable couldn't be found in either of them.
+// Note that it does not check whether a file is actually executable by current user.
+func Which(exeName string) (string, error) {
```

Comment:
For ansible, the choice so far was to hard-code the bin https://github.com/Snowflake-Labs/sansshell/blob/a76925d925410c2a6bc5e0c2da05669c12d48a4b/services/ansible/server/ansible.go#L39

do you expect us to have different shredding binary specified per user, hence we start to use `which` like approach? If so, we should probably sudo to the user first before resolving path?

(not sure if we need that, and if so we could just start with binding to `/usr/bin/shred`?)